seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -4566,10 +4566,10 @@
         ""
       ],
       "tail": [
-        "(function(x) { return \"collection = \\n\" + JSON.stringify(x, '\\n', 2); })(collection);"
+        ";(function(x) { return \"collection = \\n\" + JSON.stringify(x, '\\n', 2); })(collection);"
```

Comment:
@FreeCodeCamp/issue-moderators Also I propose all challenges to include this change in the tail code for the following reason
> Prepend semicolon in tail code IIFE so that challenge works even if challenge seed or the user code doesn't end with a semicolon. It's a protection again ASI (Automatic Semicolon Insertion). Otherwise, if the user written solution doesn't end with a semicolon, then there will be an error.